notebooks/ch10/modeling_simple.md

```diff
@@ -149,8 +149,12 @@ plt.xlabel('Tip Amount in Dollars')
 plt.ylabel('Proportion per Dollar');
 ```
 
+    /Users/captain/miniconda3/lib/python3.6/site-packages/matplotlib/axes/_axes.py:6462: UserWarning: The 'normed' kwarg is deprecated, and has been replaced by the 'density' kwarg.
+      warnings.warn("The 'normed' kwarg is deprecated, and has been "
 
-![png](modeling_simple_files/modeling_simple_5_0.png)
+
+
+![png](modeling_simple_files/modeling_simple_5_1.png)
 
 
 There are already some interesting patterns in the data. For example, there is a clear mode at \$2 and most tips seem to be in multiples of \$0.50.
```

Comment:
Hmm.. in the textbook, this show up as "\$2". Perhaps you need two backslashes?

Reply:
I didn't change that and it looks good on gitbook, so I'm gna push and see if it's okay